astro/src/content/docs/get-started/use-cases/_m2m-communication-description.mdx

```diff
@@ -0,0 +1,5 @@
+Manage authentication and permissions for software programs using the client credentials grant and access tokens. More secure than static API keys, with fine grained permissions and service identity built in. 
+
+Self-host FusionAuth and embed it into your Kubernetes cluster, on-prem data center or other environment. 
+
+FusionAuth can scale to hundreds of millions of entities (APIs, devices) and you can manage the entire solution via language specific SDKs.
```

Comment:
may want to link to blog you did on this with the performance.

Reply:
thanks! I don't typically link to blogs from docs (other way around is more common) so I think I'll leave this as is. Appreciate the suggestion.